external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaUtilsSuite.scala

```diff
@@ -61,4 +63,24 @@ class UnsafeMapSuite extends SparkFunSuite {
     assert(mapDataSer.valueArray().getInt(0) == 19286)
     assert(mapDataSer.getBaseObject.asInstanceOf[Array[Byte]].length == 1024)
   }
+
+  test("UnsafeMap from Kafka Headers") {
```

Comment:
Could you move `UnsafeMapSuite.scala` back? We should create a new test suite for `KafkaUtils.toUnsafeMapData`. The other tests should stay in `catalyst`.

Reply:
Since `UnsafeMap` is not used anymore, it would be better to remove both of `UnsafeMap` and `KafkaUtilsSuite`. If you need this class, please open a issue and notify me.